tests/meta/orders/test_meta_orders_api.py

```diff
@@ -382,13 +381,13 @@ def test_validate_sex_inconsistent_sex(
     orders_api: OrdersAPI, mip_order_to_submit: dict, helpers: StoreHelpers
 ):
     # GIVEN we have an order with a sample that is already in the database but with different sex
-    order_data = OrderIn.parse_obj(mip_order_to_submit, project=OrderType.MIP_DNA)
+    order_data = MipDnaOrder.model_validate(mip_order_to_submit)
     store = orders_api.status
     customer: Customer = store.get_customer_by_internal_id(customer_internal_id=order_data.customer)
 
     # add sample with different sex than in order
-    sample: MipDnaSample
-    for sample in order_data.samples:
+    samples: list[MipDnaSample] = [sample for _, _, sample in order_data.enumerated_new_samples]
+    for _, _, sample in order_data.enumerated_new_samples:
```

Comment:
can this be
```suggestion
    for sample in samples:
```
given line 389?

Reply:
@islean you can do this in all tests in this module